data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,1660 @@
+			`	An Unfettered herald - recognizable with his peculiar bird mask - approaches and notices you looking around. "Incredible, is it not? To see hope on the face of buildings and on the face of people. I am here to say that it's time you learn how to communicate with us - with the True language of the Hai. Not only with us. You also need to be able to make demands to the birds, and there is only one appropriate language for it. We have learned that they too speak Hai, even if they disgrace our language by doing so."`
+			choice
+				`	"How can I learn Hai?"`
+				`	"Is there some kind of translator?"`
+					goto translator
+			`	"Well to be frank, we don't have to ability to teach all of its nuances to you in time. However, we believe that our brethren do have some way of doing that, given how apt they are with languages and how keen they are to meet new races."`
+			label translator
+			`	"We think our Hai brethren may indeed have what we need, but we need to find a way to obtain it. In the meantime, You should go and meet the aliens up North and speak with them. They may not allow you to land on most of their planets. They may have grown distrusful of newcomers - probably because of our own actions. Go and see how pathethic these birds are, and then you should go to He-Yi about that translator."`
+			branch neutral
+				has "reputation: Wanderer" >= 0
+			choice
+				`	"But I am at war with them already."`
+			`	"We know. Here is a message in Hai that you will transmit to them. It says that you are our ambassador, so do not open fire on them for now. Once you have introduced yourself to the birds, and finished this mission for us, you will need to bring the Hai to this new alien race, as an ambassador. The Hai would not believe anything if it came from us. We don't want to drag our brethren into a conflict that is ours, so they need to be properly introduced, and they will certainly choose to remain neutral in it. We are the warriors, not them. But they still need to see how weak the birds are."`
+			action
+				"reputation: Wanderer" = 0
+			label neutral
+			choice
+				`	"I guess I'll try to land on their planet."`
+					accept
+				`	"I am not interested in being able to communicate with them."`
+			`	"Well if you want to work with us, you'll have to set that preference aside. And don't forget: the translator is for communicating with us as well. Some among the most conservative of us despise speaking in your language. This will be a token of good faith on your part."`
+			choice
+				`	"I guess I'll try to land on their planet."`
+					accept
+	on visit
+		dialog "You have not yet met the Wanderers and acquired a translator from <stopovers> to communicate with them. "
+	to complete
+		has "language: Wanderer"
+	on complete
+		log `The Unfettered asked me to be able to speak like a True Hai, because that is the language in which I should make demands to the Unfettered later.`
```

Comment:
To the Unfettered?

Reply:
To the wanderers right
Also to the uhai but not that much